config/environments/development.rb

```diff
@@ -15,8 +15,24 @@
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
 
-  # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  # Raise hell if the mailer can't send.
+  config.action_mailer.raise_delivery_errors = true
+
+  # Mailer settings
+  config.action_mailer.delivery_method = :smtp
+
+  config.action_mailer.smtp_settings = {
+  address: “smtp.gmail.com”,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.